lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -161,6 +164,20 @@ def compile_locale_files
         shell_cmd("rm #{miq_dir.join('locale/*/*.po')}")
         shell_cmd("rm #{miq_dir.join('locale/*.pot')}")
       end
+
+      def generate_manifests
+        FileUtils.mkdir_p(manifest_dir)
+
+        [
+          miq_dir,        "BUILD",
+          appliance_dir,  "BUILD_APPLIANCE",
+          ui_service_dir, "BUILD_UI_SERVICE",
+          ROOT_DIR,       "BUILD_RPM_BUILD"
```

Comment:
I originally built these all into a single csv file, but then realized that I wanted to split them by RPM.  Maybe it's overkill, but this is why I chose this path.

Reply:
I can change from ALL_CAPS to something else.  This started as an extension to the existing BUILD file, but these will all live in the manifest dir instead.

Reply:
@jrafanie or maybe @agrare ...would like your opinions on this.  Separate files split into RPMs, or just a single CSV in core rpm?  Or something else?

Reply:
Is the manifest for each the same as `rpm -ql ...` to list the files in the rpm?  If so, yes, I agree, manifests should match the rpm structure so 1 per rpm.

Reply:
These manifest files are basically the git shas of the repos for diagnostic... Nothing to do with rpm -q.

Reply:
Can you post an example?  I'm not sure what options are being discussed.

Reply:
```
$ cat BUILD
20220622112412_abcd1234
$ cat BUILD_APPLIANCE
20220622112412_1234abcd
...
```

vs

```csv
name,git_sha
manageiq,abcd1234
manageiq-appliance,1234abcd
...
```

Reply:
discussed on a call... it makes sense to keep the references in the rpm subpackages clean and simple so I like the idea of separate files... your first option here.